lib/frontier/feature_spec/order_expectation_method.rb

```diff
@@ -0,0 +1,25 @@
+class Frontier::FeatureSpec::OrderExpectationMethod
+
+  include Frontier::ModelConfigurationProperty
+
+  def method_name
+    "expect_#{model_configuration.as_collection}_to_be_ordered"
+  end
+
+  def to_s
+    raw = <<STRING
+def #{method_name}(first, second)
+  within(first_row)  { expect(page).to have_content(first.#{primary_attribute_name}) }
+  within(second_row) { expect(page).to have_content(second.#{primary_attribute_name}) }
+end
+STRING
+    raw.rstrip
```

Comment:
the way heredoc works...http://blog.jayfields.com/2006/12/ruby-multiline-strings-here-doc-or.html

There is a newline included between `end` on line 14 and `STRING` on line 15.

To get around that I could write `endSTRING` but that is super hard to read and text editors can't parse it. So instead I strip the newline off the right hand side of the string :)